rdmo/core/import_helpers.py

```diff
@@ -33,6 +33,8 @@ def get_value(self, **kwargs):
 
     @staticmethod
     def get_value_from_callback(callback, kwargs):
+        if not callable(callback):
```

Comment:
Why is this needed?

Reply:
to prevent (developer) mistakes in the initialization of the import helpers